core/primitives/go/math.hpp

```diff
@@ -28,4 +28,9 @@ namespace fc::primitives::go {
 namespace fc {
   using primitives::go::bigdiv;
   using primitives::go::bigmod;
+  using primitives::BigInt;
+
+  inline auto bitlen(const BigInt &x) {
+    return x ? msb(x < 0 ? -x : x) : 0;
+  }
```

Comment:
```suggestion
  using primitives::go:: bitlen;
```

Move bitlen to fc::primitives::go

Reply:
fixed